app.js

```diff
@@ -13,6 +13,28 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function getParameterByName(name) {
+  var urlParams = window.location.search;
+  name = name.replace(/[\[\]]/g, "\\$&");
+
+  var results = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)").exec(urlParams);
+  
+  if (!results) return null;
+  if (!results[2]) return '';
+  return decodeURIComponent(results[2].replace(/\+/g, " "));
+}
+
+function parseTagsInUrl() {
+  var easy = getParameterByName("easy");
+  var mentored = getParameterByName("mentored");
+  if (easy === "false" || (mentored === "true" && easy === null)) {
```

Comment:
About this logic: why do we need to even have values for `easy` and `mentored`.
It seems to me that this could work with just `?easy&mentored`, or `?easy`, or `?mentored`.
I don't think we need them to have `true/false` as a value. It makes the URL harder to type, and we can derive the same meaning with the values.

So:
* if neither `easy` nor `mentored` are part of the URL params, then we use to the default settings,
* if `easy` is in the URL params but not `mentored`, then we search for easy/non-mentored bugs 
* if `easy` is not in the URL params but `mentored` is, then we search for non-easy/mentored bugs
* if both `easy` and `mentored` are in the URL params, then we search for easy/mentored bugs

How does that sound?